autoload/kite/hover.vim

```diff
@@ -45,6 +45,11 @@ endfunction
 
 
 function! kite#hover#goto_definition_handler(response)
+  if a:response.status == 503
+    call kite#utils#warn('Locked until tomorrow (upgrade to Pro to unlock).')
```

Comment:
Can we check for and use the json message property here too, defaulting to `'unable to find a definition'` if it doesn't exist? Just cautious about the case if someone changes the route to 503 in the future for a different reason, we could forget to change this and display the wrong message. (Recently discovered this status collision on the codenav endpoint.)

Reply:
Good idea.  Please see d96ba3e.